static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -745,6 +745,14 @@ function FlamegraphZoomView({
     ]);
   }, [canvasPoolManager, flamegraph, scheduler]);
 
+  const handleCopyFunctionName = useCallback(async () => {
+    if (!hoveredNodeOnContextMenuOpen.current) {
+      return;
+    }
+
+    await navigator.clipboard.writeText(hoveredNodeOnContextMenuOpen.current.frame.name);
```

Comment:
I haven't seen it yet personally, but this function can fail if a users declines permissions.

i think this would probably only be a real issue for super paranoid users, or enterprises that have strict security protocols.

i think the UX in the past was just to toast an error. Kind of a pain; i don't see it as a blocking issue.
🤷🏼

Reply:
Oh good call, let me see if I can dispatch a toast easily if this fails.